config/crd/bases/slurm.nebius.ai_slurmclusters.yaml

```diff
@@ -1477,7 +1477,6 @@ spec:
                   defMemPerNode: 1228800
                   maxJobCount: 10000
                   minJobAge: 86400
-                  taskPlugin: task/cgroup,task/affinity
```

Comment:
Hasn't it already been removed in @rdjjke PR https://github.com/nebius/soperator/pull/277/files#diff-19296436c54bc0c51cf07e8b54a3306bd5b8cc5225d186a87fb830a3dde25d1c ?

Reply:
Yes it was, but this conflict will be resolved automatically I suppose